README.md

```diff
@@ -13,10 +13,15 @@ removes the need for a privileged, `NET_ADMIN` container in the Istio users' app
 
 ## Usage
 
-The [Istio Helm charts](https://github.com/istio/istio/tree/master/install/kubernetes/helm/istio) integrate
-the option to install the Istio CNI.  The [Istio Installation with Helm](https://preliminary.istio.io/docs/setup/kubernetes/helm-install/)
-procedure with the addition of the setting `--set istio_cni.enabled=true` enables the Istio CNI for
-the Istio installation.
+A complete set of instructions on how to use and install the Istio CNI is available on the Istio documentation sites under the "Istio Install Istio with the Istio CNI plugin".  Only a summary is provided here.  The steps are:
```

Comment:
Add a link to the page from [istio.io](https://istio.io)?

Reply:
s/sites/site

Reply:
+1 on the link to https://preliminary.istio.io/docs/setup/kubernetes/install/cni/

IMO, in this repo we should give an explicit example of installing an Istio CNI version built from source so folks can easily see how to test their changes.  So, an example in bullet 3.  Will add comment there.

Reply:
I wanted to add the link,  but since PR was still in flight I decided that it would be best to just provide a reference string to search on and then a subsequent PR can provide the proper link when the the istio side docs stabilize.   I thought putting a link to "preliminary" content was bad form.

Reply:
The preliminary link always works and the document is present, just the PR to update it is in flight.  So, IMO, use the prelim link.